source/gui/inputGestures.py

```diff
@@ -0,0 +1,798 @@
+# -*- coding: UTF-8 -*-
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2020 NV Access Limited
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+import itertools
+import re
+from typing import Tuple, Union, Dict
+
+import wx
+from wx.lib.mixins.treemixin import VirtualTree
+import wx.lib.newevent
+import gui
+from logHandler import log
+
+from typing import List, Optional
+import keyboardHandler
+from . import guiHelper
+try:
+	import updateCheck

```

Comment:
This is not used anywhere in this file. Flake8 unfortunately fails to warn about unused import if they are in try ... except block.

Reply:
Good catch, I'll remove that.